src/test/java/edu/ucla/library/iiif/fester/handlers/DeleteManifestHandlerTest.java

```diff
@@ -44,7 +44,9 @@ public void testDeleteManifestHandler(final TestContext aContext) throws IOExcep
 
             if (response.statusCode() == HTTP.SUCCESS_NO_CONTENT) {
                 aContext.assertFalse(myS3Client.doesObjectExist(myS3Bucket, myManifestS3Key));
-                asyncTask.complete();
+                if (!asyncTask.isCompleted()) {
```

Comment:
Why not just call `complete(asyncTask)`, as you do in `PutManifestHandlerTest`?

Reply:
I should call complete(asyncTask). I think I missed that because previously in this same file  
if (!asyncTask.isCompleted()) ... was placed in it. I'll change both instances of it.